scripts/aws/monitor.py

```diff
@@ -283,13 +297,17 @@ def add_cloudwatchlog_role_to_apigateway(client, role_arn):
     logging.info('Waiting for the newly created role to be available')
     # Sleep for 10 seconds to allow the role created above to be avialable for lambda function creation
     time.sleep(10)
+    attach_managed_policy(iam_client, lambda_role_name,
+                          'arn:aws:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole')
     lambda_client = session.create_client('lambda', args.aws_region)
     zip_file_name = create_lambda_function_zip(j2_env, tmpdirname, args.splunk_host,
                                                args.splunk_token, lambda_function_name)
+    vpc_config = {'SubnetIds': args.subnet_list, 'SecurityGroupIds': args.sg_list}
     create_lambda_function(lambda_client, lambda_function_name, 'nodejs4.3', lambda_exec_role_arn,
                            'index.handler', zip_file_name,
```

Comment:
As we discussed on the call, I think we need to name this lambda function differently per environment, for example, prod-edx-function, stage-edx-function.

This will mean that the script will need to know the environment, prod, and the deployment edx.

Additionally, this information will need to be passed to the role template as the function name is referenced there.

Finally, the role itself should also be named to indicate with environment and deployment it belongs to, see line 286.